common-crafting.lic

```diff
@@ -12,33 +12,49 @@ module DRCC
     ['nothing in there'].include?(DRC.bput('look in cruc', 'nothing in there', 'you see'))
   end
 
-  def find_empty_crucible
+  def find_empty_crucible(hometown)
```

Comment:
Not sure this is the place for it, but could we just have all crucible rooms ever listed, and any that are over a certain distance are ignored? It's weird that I can't use ;smith without changing my hometown.